backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5054,4 +5054,34 @@ databaseChangeLog:
             columns:
               - column:
                   name: notes
-                  type: text
\ No newline at end of file
+                  type: text
+  - changeSet:
+      id: remove-multiplex-flag
+      author: boban@skylight.digital
+      comment: inserts multiplex flag in table
+      changes:
+        - tagDatabase:
+            tag: remove-multiplex-flag
+        - delete:
+            schemaName: ${database.defaultSchemaName}
+            tableName: feature_flag
+            where: name='multiplexEnabled'
+      rollback:
+        - sql:
```

Comment:
honestly in this case, I wouldn't worry about adding it back, since we're not using it anymore. 
Going forward populating this table shouldn't be in the db migration at all. the whole idea was to be able to enable/disable features on the fly, adding it to migrations kinda defeats the purpose.

Reply:
Yeah, Joanna is working on #6492 which should resolve the issue moving forward. 
I'll just remove the rollback here though!